src/main/java/haro/Storage.java

```diff
@@ -0,0 +1,167 @@
+                    default:
+                        actionType = "none";
+                }
+
+                if (actionType == "none") {
+                    // Empty file or invalid
+                    break;
+                } else if (actionType == "todo") {
+                    try {
+                        if (storageArgs.length < 3) {
+                            throw new Exception("Invalid save list");
+                        }
+                        int markedInt = Integer.valueOf(storageArgs[1].trim());
+                        boolean marked = markedInt == 0 ? false : true;
+                        ToDo newTodo = new ToDo(storageArgs[2].trim(), marked);
+                        resultTasks.add(newTodo);
+                        continue;
+                    } catch (Exception e) {
+                        System.out.println(e.getMessage());
+                        resultTasks = new ArrayList<Task>();
+                        break;
+                    }
+                } else if (actionType == "deadline") {
+                    try {
+                        if (storageArgs.length < 4) {
+                            throw new Exception("Invalid save list");
+                        }
+
+                        int markedInt = Integer.valueOf(storageArgs[1].trim());
+                        boolean marked = markedInt == 0 ? false : true;
+                        String task = storageArgs[2].trim();
+                        String deadline = storageArgs[3].trim();
+                        Deadline newDeadline = new Deadline(task, deadline, marked);
+                        resultTasks.add(newDeadline);
+                        continue;
+
+                    } catch (Exception e) {
+                        System.out.println(e.getMessage());
+                        resultTasks = new ArrayList<Task>();
+                        break;
+                    }
+                } else if (actionType == "event") {
+                    try {
+                        if (storageArgs.length < 5) {
+                            throw new Exception("Invalid save list");
+                        }
+
+                        int markedInt = Integer.valueOf(storageArgs[1].trim());
+                        boolean marked = markedInt == 0 ? false : true;
+                        String task = storageArgs[2].trim();
+                        String start = storageArgs[3].trim();
+                        String end = storageArgs[4].trim();
+                        Event newEvent = new Event(task, start, end, marked);
+                        resultTasks.add(newEvent);
+                        continue;
+
+                    } catch (Exception e) {
+                        System.out.println(e.getMessage());
+                        resultTasks = new ArrayList<Task>();
+                        break;
+                    }
+                }
+            }
+
+        } catch (NoSuchElementException e) {
+
+        } catch (Exception e) {
+

```

Comment:
Empty catch blocks. Should a comment be included to declare that it was left empty on purpose? E.g. `// Do Nothing`